packages/ui/src/components/Checkbox/index.module.scss

```diff
@@ -0,0 +1,51 @@
+@use '@/scss/underscore' as _;
+
+.checkbox {
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+
+  .icon {
+    width: 20px;
+    height: 20px;
+    margin-right: _.unit(2);
+
+    > svg {
+      display: none;
+      color: var(--color-text-disabled);
+
+      &:first-child {
+        color: var(--color-primary);
+      }
+    }
+  }
+
+  input {
+    position: absolute;
+    width: 20px;
+    height: 20px;
+    left: 0;
+    top: 0;
+    margin: 0;
+    opacity: 0%;
+  }
+
+  input:checked:not(:disabled) ~ .icon > svg:nth-child(1),
+  input:not(:checked):not(:disabled) ~ .icon > svg:nth-child(2) {
```

Comment:
Do child1 and child2 refer to `path` and `circle` in the svg? If yes, then we can directly use them instead of nth-child.
And it looks cumbersome to define svg as `display: none` by default. Try reverse the visibility settings and it would be slightly better. e.g.
```
icon {
  svg {
    // do not set display: none;
  }
}

input:disabled ~ .icon {
  path,
  circle {
    display: none;
  }
}
```